docs/docs/launch-manual/sections/core-hacking/sections/creating-a-fork-of-a-core-package.md

```diff
@@ -0,0 +1,93 @@
+## Creating a Fork of a Core Package
+
+Several of Pulsar's core packages are maintained in the
+[`packages` directory of the pulsar-edit/pulsar repository](https://github.com/pulsar-edit/pulsar/tree/master/packages).
+If you would like to use one of these packages as a starting point for your own
+package, please follow the steps below.
+
+::: tip Tip
+
+In most cases, we recommend [generating a brand new package](#package-generator)
+or a [brand new theme](#creating-a-syntax-theme) as the starting point for your
+creation. The guide below applies only to situations where you want to create a
+package that closely resembles a core Pulsar package.
+
+:::
+
+### Creating Your New Package
+
+<!--Could this be made better with GH CLI?-->
+
+For the sake of this guide, let's assume that you want to start with the current
+code in the [one-light-ui](https://github.com/pulsar-edit/pulsar/tree/master/packages/one-light-ui)
+package, make some customizations to it, and publish your new package under the
+name "one-light-ui-plus".
+
+1. Download the [current contents of the pulsar-edit/pulsar repository as a zip file](https://github.com/pulsar-edit/pulsar/archive/master.zip)
+
+2. Unzip the file to a temporary location (for example
+   **_LNX/MAC_**: `/tmp/pulsar` -
+   **_WIN_**: `C:\TEMP\pulsar`)
+
+3. Copy the contents of the desired package into a working directory for your
+   fork
+
+::: tabs#core-hacking
+
+@tab Linux
+
+```sh
+$ cp -R /tmp/pulsar/packages/one-light-ui ~/src/one-light-ui-plus
+$ cd ~/src/one-light-ui-plus
+```
+
+@tab macOS
+
+```sh
+$ cp -R /tmp/pulsar/packages/one-light-ui ~/src/one-light-ui-plus
+$ cd ~/src/one-light-ui-plus
+
+```
+
+@tab Windows
+
+```sh
+$ xcopy C:\TEMP\pulsar\packages\one-light-ui C:\src\one-light-ui-plus /E /H /K
```

Comment:
Honestly probably yes? I assume PS comes with all versions of Windows these days and that `Copy-Item` isn't something that is only on new ones?
As for terminal, not sure, I've had it installed on all my Windows machines for ages now so not actually sure what the default is.

Reply:
I mean PowerShell even works on Linux at this point. Should be pretty safe to assume it's the modern option. I can close this if you'd like or otherwise let me whip up what should probably go in it's place. 

A suitable replacement would probably be:

```powershell
Copy-Item -Path "C:\TEMP\pulsar\packages\one-light-ui" -Destination "C:\src\one-light-ui-plus" -Recurse -Force
```

Since on XCOPY we have 
* `/E` - Copies all subdirectories, even if they are empty.
* `/H` - Copies files with hidden and system file attributes.
* `/K` - Copies files and retains the read-only attribute

Then in PowerShell we have
* `-Recurse` - Copies all subdirectories
* `-Force` - Copies hidden and system files, as well as read-only files

Reply:
```ps
$ Copy-Item -Path "C:\TEMP\pulsar\packages\one-light-ui" -Destination "C:\src\one-light-ui-plus" -Recurse
```

I think the above is correct for PS?

Reply:
Oh I was typing the above at the same time... haha

Reply:
Added this line now too